scripts/test_coverage.sh

```diff
@@ -6,4 +6,6 @@ cd $DIR/..
 python setup.py install
 
 cd pyuvdata/tests
-nosetests --with-coverage --cover-erase --cover-package=pyuvdata --cover-html "$@"
+python -m pytest --cov=pyuvdata --cov-config=../../.coveragerc\
+       --cov-report term --cov-report html:cover \
+       "$@"
```

Comment:
Will this still work the same way? For example, if I just wanted to run `test_utils.py`, I would do:
`./scripts/test_coverage.sh test_utils.py`
or to do a specific test:
`./scripts/test_coverage.sh test_utils.py:test_XYZ_from_LatLonAlt`

Reply:
to run a specific file you can still call 
`./scripts/test_coverage.sh test_utils.py`
however to run a specific test you need two `:` e.g.
`./scripts/test_coverage.sh test_utils.py::test_XYZ_from_LatLonAlt`

Reply:
I'll leave reading all the options up to users, but two common ones `-x` is the same (exit after one failure or error) and the no output capture: `-s`.